UnityProjects/MRTKDevTemplate/Packages/manifest.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "com.atteneder.gltfast": "https://github.com/atteneder/glTFast.git",
```

Comment:
It might be good to make the version we want to use explicit, by adding something like `#v4.8.3` to the end.